scripts/analysis/amwg_table.py

```diff
@@ -394,11 +420,60 @@ def _df_comp_table(adf, output_location, case_names):
     df_comp['diff'] = [f'{i:.3g}' if np.abs(i) < 1 else f'{i:.3f}' for i in diffs]
 
     #Write the comparison dataframe to a new CSV file:
-    cols_comp = ['variable', 'unit', 'test', 'control', 'diff']
+    cols_comp = ['variable', 'unit', 'test', 'baseline', 'diff']
     df_comp.to_csv(output_csv_file_comp, header=cols_comp, index=False)
 
     #Add comparison table dataframe to website (if enabled):
-    adf.add_website_data(df_comp, "Case Comparison", case_names[0], plot_type="Tables")
+    adf.add_website_data(df_comp, "case_comparison", case_names[0], plot_type="Tables")
+
+#####
+
+def _df_multi_comp_table(adf, csv_locs, case_names, test_nicknames):
+
+    main_site_path = Path(adf.get_basic_info('cam_diag_plot_loc', required=True))
+    output_csv_file_comp = main_site_path / "amwg_table_comp_all.csv"
+    df_comp = pd.DataFrame(dtype=object)
+
+    cols_comp = ['variable', 'unit']
+
+    baseline = str(csv_locs[-1])+f"/amwg_table_{case_names[-1]}.csv"
+    df_base = pd.read_csv(baseline)
+
+    for i,val in enumerate(csv_locs[:-1]): 
+        case = str(val)+f"/amwg_table_{case_names[i]}.csv"
+        df_case = pd.read_csv(case)
+        
+        if test_nicknames[i] == case_names[i]:
+            df_comp[['variable','unit',f"case {i+1}"]] = df_case[['variable','unit','mean']]
+            cols_comp.append(f"case {i+1}")
+        else:
+            df_comp[['variable','unit',f"{test_nicknames[i]}"]] = df_case[['variable','unit','mean']]
+            cols_comp.append(test_nicknames[i])
```

Comment:
Essentially this is to check if the case name is used (or whether the test_nicknames are not assigned and defaults to case name) to populate table header. If so because the case name is generally pretty long, it just shortens the header to "case<#> " to keep the table from being very long, especially if the number of cases is getting large (~ >5).

Otherwise, if actual nicknames are used (assuming they aren't nearly as long as raw case names), the column header uses them.